pkg/controller/logstorage/linseed/eskubecontrollers.go

```diff
@@ -72,30 +90,32 @@ func (r *ReconcileLogStorage) createESKubeControllers(
 		Authentication:               authentication,
 		KubeControllersGatewaySecret: kubeControllersUserSecret,
 		LogStorageExists:             true,
-		TrustedBundle:                certificateManager.CreateTrustedBundle(),
+		TrustedBundle:                certificateManager.CreateTrustedBundle(), // We don't care about the contents.
+		Namespace:                    request.InstallNamespace(),               // TODO: This will give tigera-elasticsearch in single-tenant systems. Is this OK?
```

Comment:
Currently, es-kube-controllers runs in calico-system. I've always thought this was a little weird...

a side-effect of what I'm doing here is to run it in tigera-elasticsearch (for single-tenant clusters). I think this is OK? But would like input.

If we do manage to get rid of es-kube-controllers entirely for multi-tenant management clusters, we could more easily keep it in calico-system and just simply not install it for multi-tenant setups.

Reply:
I'm not sure if you saw my response on the PR comment section, but I posted this:

> The reason es kube controllers is in the calico system namespace is so that it would have permissions in the managed clusters, as it uses the regular kube controllers service account.
>
> If we're to separate this out, you need to make sure the managed clusters have the SA for es kubecontrollers / RBAC, which would require an upgrade to the managed clusters.